core/src/main/java/com/netflix/conductor/core/execution/tasks/SystemTaskWorkerCoordinator.java

```diff
@@ -126,8 +146,10 @@ private void listen() {
 	}
 	
 	private void listen(WorkflowSystemTask systemTask) {
-		Executors.newScheduledThreadPool(1).scheduleWithFixedDelay(()->pollAndExecute(systemTask), 1000, pollFrequency, TimeUnit.MILLISECONDS);
+		ScheduledExecutorService taskPool = Executors.newScheduledThreadPool(1);
+		taskPool.scheduleWithFixedDelay(()->pollAndExecute(systemTask), 1000, pollFrequency, TimeUnit.MILLISECONDS);
 		logger.debug("Started listening {}", systemTask.getName());
+		taskPools.put(systemTask.getName(), taskPool);
```

Comment:
I did not find where taskPools is cleared. It seems that it can cause OutOfMemory